frontend/app_flowy/packages/flowy_infra/lib/theme.dart

```diff
@@ -93,9 +93,9 @@ class AppTheme {
 
       case ThemeType.dark:
         return AppTheme(isDark: true)
-          ..surface = _black
-          ..hover = _main2
-          ..selector = _black
+          ..surface = const Color(0xff292929)
```

Comment:
This file has a warning on line 13. Please fix.

Reply:
Removed the warnings